### DISCUSSION TITLE
Font problem

### DISCUSSION THREAD

**Discussion by MIMO-PIESEK** [elementary/triage#669] [Q&A]
hey, I have a problem with the font in elementary os 8, everything was great but the font is unreadable and I don't know what it says. please help - I donated to elementaryOs
screen shots:
![Zrzut ekranu z 2024-12-30 11 19 36](https://github.com/user-attachments/assets/0d9d8d59-7f1d-4adf-98d8-53f620508d00)

**Comment by ryonakano** [comment]
We're sorry for your inconvenience. First of all, this looks like a issue related to your graphics driver, so I'm not sure we can help you.

What kind of hardware are you using?

**Comment by teamcons** [comment]
Try a different NVIDIA driver. Try tutorials for nvidia on Ubuntu. I think you need to add a boot parameter?
They are a pain to set up
One of the tab in that page is for drivers by the way

**Comment by Alcizmax** [comment]
@MIMO-PIESEK I'm having this same problem when browsing apps in the central app store. 
![Captura de tela de 2025-01-05 02 52 59](https://github.com/user-attachments/assets/c5dc905e-8e7b-46ef-b643-206116f4137a)

**Comment by Alcizmax** [comment]
![Captura de tela de 2025-01-05 02 53 05](https://github.com/user-attachments/assets/37075c7a-9650-47d9-ae5b-bcea782f3644)
